File: plugins/devkit/devkit-java-tests/testData/inspections/inspectionDescription/MyInspectionCustomShortName.java
import com.intellij.codeInspection.InspectionProfileEntry;

public class <error descr="Inspection does not have a description [getShortName()]">MyInspectionCustomShortName</error> extends InspectionProfileEntry {

  public String getShortName() {
    return "NOT_EXISTING_CUSTOM_SHORT_NAME";
  }
}

File: plugins/devkit/devkit-kotlin-tests/testData/inspections/inspectionDescription/MyInspectionCustomShortName.kt
class <error descr="Inspection does not have a description [getShortName()]">MyInspectionCustomShortName</error> : com.intellij.codeInspection.InspectionProfileEntry() {
  override fun getShortName(): String {
    return "NOT_EXISTING_CUSTOM_SHORT_NAME"
  }
}
